src/pages/students/Students.tsx

```diff
@@ -1,11 +1,192 @@
-import './Students.css'
+import "./Students.css";
+import { useForm } from "react-hook-form";
+import type { Resolver } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { studentsScheme } from "../students/students.scheme";
+import { MOCKED_PAYMENT_TYPES } from "../../mocks/paymentTypes";
+import { useNavigate } from "react-router-dom";
+import ErrorIcon from "../../assets/error-icon.png";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { Controller } from "react-hook-form";
 
+type FormData = {
+  name: string;
+  lastname: string;
+  nroTelefono: string;
+  fechaNacimiento: Date | null;
+  tipoPago: string;
+  cantidadDias: number;
+  diaPago: string;
+  diasExtra: string;
+  fechaIngreso: Date | null;
+};
 function Students() {
+  const navigate = useNavigate();
+  const resolver = yupResolver(studentsScheme) as Resolver<FormData>;
+
+  const onSubmit = (data: any) => {
+    console.log("Formulario enviado con datos:", data);
+  };
+  const {
+    register,
+    handleSubmit,
+    watch,
+    control,
+    formState: { errors },
+  } = useForm<FormData>({
+    resolver,
+    defaultValues: {
+      name: "",
+      lastname: "",
+      nroTelefono: "",
+      fechaNacimiento: null,
+      tipoPago: "",
+      diaPago: "",
+      diasExtra: "",
+      fechaIngreso: null,
+    },
+  });
+  const tipoPagoSeleccionado = watch("tipoPago");
   return (
-    <div>
-      <p>Students</p>
-    </div>
-  )
+    <form className="form-container" onSubmit={handleSubmit(onSubmit)}>
+      <div className="form-group">
+        <div className="column1">
+          <input {...register("name")} placeholder="Nombre" />
+          {errors.name && (
+            <div className="error-container">
+              <img src={ErrorIcon} alt="Error" />
+              <p className="error-message">{errors.name.message}</p>
+            </div>
```

Comment:
Todo este codigo, se repite mucho (por cada field), podrias armar un componente custom para esto

```
type ErrorMessageProps = {
  error?: FieldError;
};

export const ErrorMessage = ({ error }: ErrorMessageProps) => {
  return (
    <>
      {
        error && (
          <div className="error-container">
            <img src={ErrorIcon} alt="Error" />
            <p className="error-message">{error.message}</p>
          </div>
        )
      }
    </>
  )
}
```

Y aca lo modificas asi

```suggestion
          <ErrorMessage error={errors.name} />
```